backend_new/src/dtos/listings/listing-utility-create.dto.ts

```diff
@@ -1,8 +1,4 @@
 import { OmitType } from '@nestjs/swagger';
 import { ListingUtilities } from './listing-utility.dto';
 
-export class ListingUtilitiesCreate extends OmitType(ListingUtilities, [
```

Comment:
same as above 
"since we aren't making changes to the ListingUtilities anymore I think we should just drop ListingUtilitiesCreate and leverage ListingUtilities in those spot"

Reply:
👍 I've deleted this DTO